class/defaults.yml

```diff
@@ -1,3 +1,9 @@
 parameters:
   csi_driver_smb:
-    namespace: syn-csi-driver-smb
+    namespace: kube-system
```

Comment:
Is there a technical reason for deploying the driver in the `kube-system` namespace?

Reply:
Not really sure. I'm just following the recommendation / guide listed here: https://github.com/kubernetes-csi/csi-driver-smb/tree/master/charts

Reply:
Ah, it's probably because the pods default to priority class `system-cluster-critical` which is non-trivial to make available outside the `kube-system` namespace.

Reply:
Ah right. So OK for you to leave it as is and finally merge this?